jobbergate-api/dev_tools/db.py

```diff
@@ -75,16 +77,8 @@ def migrate(
     Create alembic migrations for a local database.
     """
     logger.debug(f"Creating migration with message: {message}")
-    commands = [
-        "alembic",
-        "--config=alembic/alembic.ini",
-        "revision",
-        f"--message={message}",
-    ]
-    if not blank:
-        commands.append("--autogenerate")
-
-    subprocess.run(commands)
+    config = Config(file_="alembic/alembic.ini")
+    sqla_migrate(config, message=message, autogenerate=not blank)
```

Comment:
This is nice. Just to be sure, is the functionality equivalent?

Reply:
Yeah, it's literally what the Alembic CLI calls directly. The difference is that with this, you don't do it in a subshell, so you can see the alembic output as it runs. Also, it's cleaner.